### ISSUE TITLE
Accessibility testing v3.0.11: For you tab labels 

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#565]
For both iOS and Android: 

- Edit (button label missing), should be Edit in English and Modifier in French
- See more (button label missing), should be See more in English and Afficher plus in French 
- "..." more options button - label missing inside a subject screen, shoudl be More options in English and Plus d'options in French

For iOS: 
The subject name should be labelled as heading such as "Agriculture and food- heading"

**Comment by weglmar** [comment]
Seems to be resolved. 
@Rajat-Sen, could you please confirm?

**Comment by Rajat-Sen** [comment]
Fixed on iOS.

Will need to validate on Android.

**Comment by Rajat-Sen** [comment]
@weglmar, Validated on Android as well.
